IdentityCore/src/MSIDBrokerConstants.m

```diff
@@ -26,8 +26,13 @@
 NSString *const MSID_BROKER_ADAL_SCHEME            = @"msauth";
 NSString *const MSID_BROKER_MSAL_SCHEME            = @"msauthv2";
 NSString *const MSID_BROKER_NONCE_SCHEME           = @"msauthv3";
-NSString *const MSID_BROKER_APP_BUNDLE_ID          = @"com.microsoft.azureauthenticator";
-NSString *const MSID_BROKER_APP_BUNDLE_ID_DF       = @"com.microsoft.azureauthenticator-df";
+#if TARGET_OS_IPHONE && !TARGET_OS_MACCATALYST
+    NSString *const MSID_BROKER_APP_BUNDLE_ID          = @"com.microsoft.azureauthenticator";
+    NSString *const MSID_BROKER_APP_BUNDLE_ID_DF       = @"com.microsoft.azureauthenticator-df";
+#elif TARGET_OS_OSX
+    NSString *const MSID_BROKER_APP_BUNDLE_ID          = @"com.microsoft.CompanyPortalMac";
+    NSString *const MSID_BROKER_APP_BUNDLE_ID_DF       = @"com.microsoft.CompanyPortalMac";
```

Comment:
Should dogfood bundle identifier be different?

Reply:
My understanding, based on my discussion last Friday with you & Dan, was that iOS should have different dogfood bundle id, while Mac OS has the same for both regular and dogfood. 

Below is the values that I have captured. 

iOS: "com.microsoft.azureauthenticator"
macOS: "com.microsoft.identity.ssoextension"
 
Bundle ID:
iOS: "com.microsoft.azureauthenticator" and "com.microsoft.azureauthenticator-df"
macOS: "com.microsoft.CompanyPortalMac"
 
App group:
iOS: "group.com.microsoft.azureauthenticator.sso"
macOS: "com.microsoft.identity.ssoextensiongroup"

Reply:
Makes sense, thanks!